README.md

```diff
@@ -1588,3 +1588,273 @@ peer chaincode invoke -n mycc -c '{"Args":["queryObjectiveLeaderboard","{\"objec
  ]
 }
 ```
+Smart contract: `queryComputePlan`
+
+##### JSON Inputs:
+```go
+{
+ "key": string (required,len=64,hexadecimal),
+}
+```
+##### Command peer example:
+```bash
+peer chaincode invoke -n mycc -c '{"Args":["queryComputePlan","{\"key\":\"432fcffdf68892f5e4adeeed8bb618beaeaecf709f840671eca724a3e3109369\"}"]}' -C myc
+```
+##### Command output:
+```json
+{
+ "algoKey": "fd1bb7c31f62244c0f3a761cc168804227115793d01c270021fe3f7935482dcc",
+ "computePlanID": "432fcffdf68892f5e4adeeed8bb618beaeaecf709f840671eca724a3e3109369",
+ "objectiveKey": "5c1d9cd1c2c1082dde0921b56d11030c81f62fbb51932758b58ac2569dd0b379",
+ "outputTesttuples": [
```

Comment:
The json key should probably be `testuples` not `outputTesttuples`

Reply:
@thibaultrobert @jmorel could you please confirm this is the desired name?

Reply:
Yes, absolutely!

Reply:
Fixed https://github.com/SubstraFoundation/substra-chaincode/pull/22/commits/8dde5f477396eecd715d5372a99b2bc069623586